.github/workflows/dart_ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         # Can't run on `stable` (Dart 3) until we're fully null-safe.
-        sdk: [2.18.7]
+        sdk: [2.18.7, 2.19.6]
```

Comment:
Is there a reason to keep 2.18.7?

Reply:
Mm good catch. I don't think there is.

Though, we do currently format under 2.18.7, so we'd have to update that to run on 2.19. To avoid potential merge conflicts with the v7_wip branch, I'd vote to wait to remove it and switch formatting over until after that's merged.

Reply:
sure